fe/src/main/java/org/apache/doris/metric/MetricRepo.java

```diff
@@ -182,13 +187,24 @@ public Long getValue() {
         COUNTER_IMAGE_PUSH = new LongCounterMetric("image_push",
                 "counter of image succeeded in pushing to other frontends");
         PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_IMAGE_PUSH);
-        COUNTER_TXN_SUCCESS = new LongCounterMetric("txn_success",
-                "counter of success transactions");
+
+        COUNTER_TXN_BEGIN = new LongCounterMetric("txn_begin", "counter of begining transactions");
+        PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_TXN_BEGIN);
+        COUNTER_TXN_FAILED = new LongCounterMetric("txn_failed", "counter of failed transactions");
+        COUNTER_TXN_SUCCESS = new LongCounterMetric("txn_success", "counter of success transactions");
         PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_TXN_SUCCESS);
-        COUNTER_TXN_FAILED = new LongCounterMetric("txn_failed",
-                "counter of failed transactions");
+        COUNTER_TXN_FAILED = new LongCounterMetric("txn_failed", "counter of failed transactions");
         PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_TXN_FAILED);
 
+        COUNTER_ROUTINE_LOAD_ROWS = new LongCounterMetric("routine_load_rows", "total rows of routine load");
+        PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_ROUTINE_LOAD_ROWS);
+        COUNTER_ROUTINE_LOAD_RECEIVED_BYTES = new LongCounterMetric("routine_load_receive_bytes",
+                "total received bytes of routine load");
+        PALO_METRIC_REGISTER.addPaloMetrics(COUNTER_ROUTINE_LOAD_RECEIVED_BYTES);
+        COUNTER_ROUTINE_LOAD_ERROR_ROWS = new LongCounterMetric("routine_load_error_rows",
```

Comment:
Is it meaningful for our or user without group by job id? Or maybe `COUNTER_ROUTINE_LOAD_ROWS ` + `COUNTER_ROUTINE_LOAD_ERROR_ROWS ` is more meaningful?

Reply:
I think each job's loaded rows or error rows should be saved in each job.
This metrics are for a global perspective of system runtime overview.
I use COUNTER_ROUTINE_LOAD_ERROR_ROWS to monitor if there are too many error rows.